### DISCUSSION TITLE
Skipping some traces

### DISCUSSION THREAD

**Discussion by cemo** [open-telemetry/opentelemetry-java-instrumentation#1958] [Q&A]
We would like to customise and drop some of our traces. Especially the ones for health-check or other system related ones. How can I do it?

**Comment by iNikem** [comment]
@cemo As I have already answered you in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1060, such functionality is not yet implemented. PRs are welcomed! :) That issue contains the explanation of what should be done.
